src/Commands/ContinuousIntegration/TravisCi/TravisCiQuestionProxy.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace QuickStrap\Commands\ContinuousIntegration\TravisCi;
+
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Question\Question;
+
+class TravisCiQuestionProxy
```

Comment:
think it is possible to override the question helper for a single command? it looks like helpers are pretty much hard-wired to be shared via Application constructor. ah well.